src/main/java/controller/DeleteProductServlet.java

```diff
@@ -0,0 +1,27 @@
+package controller;
+
+import factory.ProductServiceFactory;
+import service.ProductService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(value = "/products/delete")
+public class DeleteProductServlet extends HttpServlet {
+
+    private static final ProductService productService = ProductServiceFactory.getProductService();
+
+    @Override
+    protected void doGet(HttpServletRequest request,
+                         HttpServletResponse response) throws ServletException, IOException {
+        String id = request.getParameter("id");
+        if (id != null) {
+            productService.deleteProduct(Long.valueOf(id));
+        }
```

Comment:
do we need redirect here?

Reply:
yes, when i change something, i deleted it